tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1748,29 +1757,44 @@ def test_large_generation_multilingual(self):
             torch_device
         )
 
+        # Japanese transcription, batch size 1
         generated_ids = model.generate(
             input_features, do_sample=False, max_length=20, language="<|ja|>", task="transcribe"
         )
         transcript = processor.batch_decode(generated_ids, skip_special_tokens=True)[0]
 
-        EXPECTED_TRANSCRIPT = "木村さんに電話を貸してもらいました"
-        self.assertEqual(transcript, EXPECTED_TRANSCRIPT)
+        EXPECTED_TRANSCRIPT_JA = "木村さんに電話を貸してもらいました"
+        self.assertEqual(transcript, EXPECTED_TRANSCRIPT_JA)
 
+        # English transcription, batch size 1
         generated_ids = model.generate(
             input_features, do_sample=False, max_length=20, language="<|en|>", task="transcribe"
         )
         transcript = processor.batch_decode(generated_ids, skip_special_tokens=True)[0]
 
-        EXPECTED_TRANSCRIPT = " Kimura-san called me."
-        self.assertEqual(transcript, EXPECTED_TRANSCRIPT)
+        EXPECTED_TRANSCRIPT_EN = " Kimura-san called me."
+        self.assertEqual(transcript, EXPECTED_TRANSCRIPT_EN)
 
+        # Both languages in the same batch
```

Comment:
Nice test! Would you mind defining a new test function for this test and moving the new checks into here? (e.g. `test_large_generation_batched_languages`) Keeping the tests small and modular helps identify which parts of the code are potentially broken with new PRs

Reply:
Done!